homework/index.js

```diff
@@ -30,18 +30,94 @@
     return elem;
   }
 
+  function render(parent, repositories) {
+    const table = createAndAppend('table', parent, {});
+    const tbody = createAndAppend('tbody', table, {});
+    const tr = createAndAppend('tr', tbody);
+    createAndAppend('td', tr, { class: 'label', text: 'Repository : ' });
+    const td = createAndAppend('td', tr);
+    createAndAppend('a', td, {
+      text: repositories.name,
+      href: 'https://github.com/HackYourFuture/alumni',
+      target: '_blank',
+    });
+    const tr1 = createAndAppend('tr', tbody);
+    createAndAppend('td', tr1, { text: 'Description: ', class: 'label' });
+    createAndAppend('td', tr1, {
+      text: repositories.description,
+    });
+    const tr2 = createAndAppend('tr', tbody);
+    createAndAppend('td', tr2, { text: 'Forks :', class: 'label' });
+    createAndAppend('td', tr2, {
+      text: repositories.forks_count,
+    });
+    const tr3 = createAndAppend('tr', tbody);
+    createAndAppend('td', tr3, { text: 'Updated :', class: 'label' });
+    createAndAppend('td', tr3, { text: repositories.updated_at });
+  }
+
+  function listOfContributors(contributors, parent) {
+    createAndAppend('h3', parent, { class: ' contributor - header', text: 'contributions' });
+    const ul = createAndAppend('ul', parent, { class: 'contributor-list' });
+    contributors.forEach(x => {
```

Comment:
`x` 🤔.
Having a good names, having a good context, as much your names are readable as much your code is maintainable.